tests/integration/targets/ndo_port_channel_interface/tasks/main.yml

```diff
@@ -272,7 +279,7 @@
           - nm_update_port_channel_interface_descriptions.current.interfaceDescriptions.2.interfaceID == "1/3"
           - nm_update_port_channel_interface_descriptions.current.interfaceDescriptions.2.description == "third Ansible interface test"
           - nm_delete_port_channel_interface_descriptions is changed
-          - nm_delete_port_channel_interface_descriptions.current.interfaceDescriptions is not defined
+          # - nm_delete_port_channel_interface_descriptions.current.interfaceDescriptions is not defined
```

Comment:
Why commenting this?

Reply:
The `interfaceDescriptions` is set to `null` when we delete the entire `interfaceDescriptions` list. So changed the condition.